src/clr-addons/history/history.scss

```diff
@@ -0,0 +1,21 @@
+@include exports('history.clr-addons') {
```

Comment:
this whole file should be merged with history-pinned, as it defines rules for the pinned component